### ISSUE TITLE
Feature proposal: Screensaver enhancements

### ISSUE THREAD

**Issue by kpfleming** [hrvach/deskhop#66]
(I'm happy to submit a fully-tested PR for this, just want to ensure that the features are something that are acceptable first).

* Enable screensaver independently for A and B outputs
* Enable screensaver only when an output is not the active output
* Disable screensaver after a maximum amount of time

**Comment by hrvach** [comment]
Yes, all of those are good ideas and I'll be more than happy to merge them! I'm very grateful for any help and PRs. :)
